Vulkan headers are now included unconditionally

```
In file included from ../video/out/vulkan/formats.h:4:0,
                 from ../video/out/opengl/hwdec_cuda.c:38:
../video/out/vulkan/common.h:26:10: fatal error: vulkan/vulkan.h: No such file or directory
 #include <vulkan/vulkan.h>
          ^~~~~~~~~~~~~~~~~
compilation terminated.
```
It should be possible to compile hwdec_cuda.c without vulkan headers as before.

--- a/video/out/opengl/hwdec_cuda.c
+++ b/video/out/opengl/hwdec_cuda.c
@@ -32,12 +32,17 @@
 #include <libavutil/hwcontext_cuda.h>
 
 #include "video/out/gpu/hwdec.h"
+#include "video/out/gpu/utils.h"
 #include "formats.h"
 #include "options/m_config.h"
+#if HAVE_GL
 #include "ra_gl.h"
+#endif
+#if HAVE_VULKAN
 #include "video/out/vulkan/formats.h"
 #include "video/out/vulkan/ra_vk.h"
 #include "video/out/vulkan/utils.h"
+#endif
 
 #if HAVE_WIN32_DESKTOP
 #include <versionhelpers.h>

